gallery/src/components/NxProgressBar/NxProgressBarExample.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxProgressBar } from '@sonatype/react-shared-components';
+
+export default function NxProgressBarExample() {
+  return (
+    <>
+      <div>
+        <NxProgressBar max={50}
```

Comment:
This one should have some surrounding inline content to demonstrate its alignment.  It is also missing the horizontal margin specified in sketch.
![image](https://user-images.githubusercontent.com/3630091/159475739-841b0eaf-96a5-4836-9752-3e2dbe4bed68.png)

Reply:
> This one should have some surrounding inline content to demonstrate its alignment. It is also missing the horizontal margin specified in sketch. ![image](https://user-images.githubusercontent.com/3630091/159475739-841b0eaf-96a5-4836-9752-3e2dbe4bed68.png)

Fixed as part of this commit: d2db7cf70a0250f16e446e9ff7882131108e315e

Reply:
> Fixed as part of this commit: https://github.com/sonatype/sonatype-react-shared-components/commit/d2db7cf70a0250f16e446e9ff7882131108e315e

Come to think of it, we probably need a mockup from design to demonstrate exactly what the correct vertical alignment of inline progress bars relative to text is.  What you have now might be good but I'm not certain.

Reply:
> > Fixed as part of this commit: [d2db7cf](https://github.com/sonatype/sonatype-react-shared-components/commit/d2db7cf70a0250f16e446e9ff7882131108e315e)
> 
> Come to think of it, we probably need a mockup from design to demonstrate exactly what the correct vertical alignment of inline progress bars relative to text is. What you have now might be good but I'm not certain.

I'll reach out to Jason!

Reply:
> > Fixed as part of this commit: [d2db7cf](https://github.com/sonatype/sonatype-react-shared-components/commit/d2db7cf70a0250f16e446e9ff7882131108e315e)
> 
> Come to think of it, we probably need a mockup from design to demonstrate exactly what the correct vertical alignment of inline progress bars relative to text is. What you have now might be good but I'm not certain.

I talked to Jason, and they confirmed that it will be how it is in the example right now. They are working on updating the Sketch file 👍